tests/builder/main.rs

```diff
@@ -37,6 +37,7 @@ mod opts;
 mod positionals;
 mod posix_compatible;
 mod possible_values;
+mod possible_values_expensive;
```

Comment:
Please move this into the regular `possible_values` mod.

To keep tests discoverable / mangeable, we keep all tests along the same theme in one file.

Reply:
Would it be acceptable to move it into the same file but keep it as a separate module? Then I can have the `cfg` on the module as a whole and not each individual member.

I'll push a commit shortly demonstrating the proposed change, but I'll want to squash it as a fixup before merge, so let me know.

Reply:
# I can understand that pain and a `mod` is fine.

Reply:
This should now be resolved. Leaving the thread open to ensure you agree before merge.

(To be honest, I find the 'resolve conversation' feature of github/gitlab to be confusing in this regard… I never really understood how the maintainer is supposed to easily verify that the requested changes were made – not without going through every thread. But maybe that's just me 🙂)